spec/command/ps_stop_spec.rb

```diff
@@ -9,29 +9,58 @@
     run_cpl_command!("ps:start", "-a", app, "--wait")
   end
 
-  it "stops all workloads", :slow do
-    result = run_cpl_command("ps:stop", "-a", app)
+  context "when no workload is provided" do
+    it "stops all workloads", :slow do
+      result = run_cpl_command("ps:stop", "-a", app)
 
-    expect(result[:status]).to eq(0)
-    expect(result[:stderr]).to match(/Stopping workload 'rails'[.]+? done!/)
-    expect(result[:stderr]).to match(/Stopping workload 'postgres'[.]+? done!/)
+      expect(result[:status]).to eq(0)
+      expect(result[:stderr]).to match(/Stopping workload 'rails'[.]+? done!/)
+      expect(result[:stderr]).to match(/Stopping workload 'postgres'[.]+? done!/)
+    end
+
+    it "stops all workloads and waits for them to not be ready", :slow do
+      result = run_cpl_command("ps:stop", "-a", app, "--wait")
+
+      expect(result[:status]).to eq(0)
+      expect(result[:stderr]).to match(/Stopping workload 'rails'[.]+? done!/)
+      expect(result[:stderr]).to match(/Stopping workload 'postgres'[.]+? done!/)
+      expect(result[:stderr]).to match(/Waiting for workload 'rails' to not be ready[.]+? done!/)
+      expect(result[:stderr]).to match(/Waiting for workload 'postgres' to not be ready[.]+? done!/)
+    end
   end
 
-  it "stops specific workload", :slow do
-    result = run_cpl_command("ps:stop", "-a", app, "--workload", "rails")
+  context "when workload is provided" do
+    it "stops specific workload", :slow do
+      result = run_cpl_command("ps:stop", "-a", app, "--workload", "rails")
 
-    expect(result[:status]).to eq(0)
-    expect(result[:stderr]).to match(/Stopping workload 'rails'[.]+? done!/)
-    expect(result[:stderr]).not_to include("postgres")
+      expect(result[:status]).to eq(0)
+      expect(result[:stderr]).to match(/Stopping workload 'rails'[.]+? done!/)
+      expect(result[:stderr]).not_to include("postgres")
+    end
   end
 
-  it "stops all workloads and waits for them to not be ready", :slow do
-    result = run_cpl_command("ps:stop", "-a", app, "--wait")
+  context "when replica is provided" do
+    let!(:replica) do
+      run_cpl_command!("ps:stop", "-a", app, "--wait")
+      run_cpl_command!("ps:start", "-a", app, "--wait")
```

Comment:
Need to stop and start to make sure there aren't any leftover replicas when running `ps` below.